src/Microsoft.PowerShell.Commands.Utility/commands/utility/Measure-Object.cs

```diff
@@ -631,13 +630,15 @@ private object Compare(object objValue, object statMinOrMaxValue, bool isMin)
                 statValue = PSObject.AsPSObject(statValue).ToString();
             }
 
-            if ((statValue == null) ||
-                ((LanguagePrimitives.Compare(statValue, currentValue, false, CultureInfo.CurrentCulture) * factor) > 0))
+            if (statValue == null)
             {
                 return objValue;
             }
 
-            return statMinOrMaxValue;
+            int comp = LanguagePrimitives.Compare(statValue, currentValue, false, CultureInfo.CurrentCulture);
```

Comment:
Please use more informative name instead of `comp` and add name for `false` (since we changed the code).

Reply:
`comparisonResult`?